### ISSUE TITLE
tests/manpage-scan.pl is missing from 7.49.0 tarball

### ISSUE THREAD

**Issue by atsampson** [curl/curl#814]
Tests 1139 and 1140 fail in curl 7.49.0, because the `tests/manpage-scan.pl` and `tests/nroff-scan.pl` scripts weren't included in the tarball. They need adding to `Makefile.am`.

**Comment by kdudka** [comment]
Thanks for the report!  This seems to be already fixed via 54e4c6c3.

**Comment by jay** [comment]
Thanks. Maybe we should leave this open for a while in case anyone else has the same issue today they'll see it rather than opening a new report.

You can get the dependencies like this:
~~~
cd tests
curl -fLO https://raw.githubusercontent.com/curl/curl/curl-7_49_0/tests/manpage-scan.pl
curl -fLO https://raw.githubusercontent.com/curl/curl/curl-7_49_0/tests/nroff-scan.pl
./runtests.pl 1139 1140
~~~

1140 will still fail due to a missing manpage, see #816

**Comment by bagder** [comment]
Fixed in the new 7.49.1 tarball!
